nautobot/docs/additional-features/jobs.md

```diff
@@ -349,6 +349,12 @@ http://nautobot/api/extras/jobs/local/example/MyJobWithVars/run/ \
 --data '{"data": {"foo": "somevalue", "bar": 123}, "commit": true}'
 ```
 
+The URL contains the `class_path` element that is composed of 3 elements, from the above example:
+
+- `local` or `plugin` - depending on where the `Job` has been defined.
+- `example` - path to the job definition file, for example: `my_plugin_name.jobs.my_job_filename`.
```

Comment:
```suggestion
- `example` - path to the job definition file; in this example, a locally installed `example.py` file. For a plugin-provided job, this might be something like `my_plugin_name.jobs.my_job_filename`.
```